modules/media/jsx/mediaIndex.js

```diff
@@ -60,6 +61,22 @@ class MediaIndex extends Component {
       });
   }
 
+  /**
+   * Refresh table data after successful file upload
+   * and display recent data.
+   */
+  refreshData() {
+    fetch(this.props.dataURL, {credentials: 'same-origin'})
+      .then((resp) => resp.json())
+      .then((data) => this.setState({
+        data: data.Data,
+      }))
+      .catch((error) => {
+        this.setState({error: true});
+        console.error(error);
+      });
+  }
+
```

Comment:
Personally, this seems a bit redundant and I think `fetchData` could just be used again.

Reply:
@HenriRabalais reason I chose not to use fetchData is that the user filters would be reset (I assume) if any user input existed.